data/json/items/migration.json

```diff
@@ -149,5 +149,11 @@
     "type": "MIGRATION",
     "replace": "jar_glass_sealed",
     "contents": [ "tomato_canned" ]
+  },
+  {
+    "id": "jar_kompot",
```

Comment:
You would need to write a `jq` script and considering `n=21` I'd just do them by hand which should take no more than a few minutes.

Clean and well organised JSON is as important as good C++ code and we have historically done very badly at the former.